motoman_driver/MotoPlus/_buildLog/MotoROSDX100.log

```diff
@@ -1,8 +1,8 @@
-﻿
+﻿EXEC : warning : TED - Don't forget to remove the hardware-key short circuit...
```

Comment:
Is this a TODO that's still relevant?

Reply:
Oops. 
That doesn't actually affect the build. So, I have just ignored the buildlog files.